bin.ts

```diff
@@ -60,6 +60,11 @@ const argv = yargs(hideBin(process.argv))
     alias: 'url',
     describe: 'The url to your Sentry installation\nenv: SENTRY_WIZARD_URL',
   })
+  .option('saas', {
```

Comment:
Realized we didn't add `--project` and `--org` to this list. @obostjancic was this on purpose? If not, I'll quickly add it. This ensures that `--help` lists the two options in the help message.

Reply:
I also noticed that it's missing from the readme file

Reply:
Good point. I think the readme was generally a bit outdated. I updated it, along with adding `--project` and `--org` to the readme and `--help` output in a follow-up PR: https://github.com/getsentry/sentry-wizard/pull/679